docs/extras/modules/data_connection/vectorstores/integrations/meilisearch.ipynb

```diff
@@ -68,87 +91,60 @@
     "import os\n",
     "import getpass\n",
     "\n",
+    "# Prompt for Meilisearch credentials\n",
+    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")\n",
+    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")\n",
+    "\n",
+    "# Prompt for other third-party credentials\n",
     "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"OpenAI API Key:\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you don't want to have to send your client or Meilisearch instance address, you can directly configure the [environment variables](https://www.meilisearch.com/docs/learn/configuration/instance_options#environment-variables)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")"
+    "### Adding text and embeddings\n",
+    "\n",
+    "This example adds text to the Meilisearch vector store along with the associated embeddings."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Add texts with user-friendly interface"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "from langchain.vectorstores import Meilisearch\n",
     "from langchain.embeddings.openai import OpenAIEmbeddings\n",
     "from langchain.text_splitter import CharacterTextSplitter\n",
     "\n",
-    "embeddings = OpenAIEmbeddings()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "# OpenAI API key is loaded from environment\n",
+    "embeddings = OpenAIEmbeddings()\n",
+    "\n",
+    "# Read the file content\n",
```

Comment:
```suggestion
```
Same.Here, these lines of code are already explained in
in the documentation [here](https://python.langchain.com/docs/modules/data_connection/document_transformers/text_splitters/character_text_splitter)